osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -407,11 +407,11 @@ private double getScore(ScoringMode mode)
             {
                 default:
                 case ScoringMode.Standardised:
-                    return max_score * (base_portion * baseScore / maxBaseScore + combo_portion * HighestCombo.Value / maxHighestCombo) + bonusScore;
+                    return (max_score * (base_portion * baseScore / maxBaseScore + combo_portion * HighestCombo.Value / maxHighestCombo) + bonusScore) * scoreMultiplier;
```

Comment:
not sure we want to apply the multiplier to bonus score, but should be ok for now.

Reply:
that's the way it worked before for **standardised**, is there an algorithm for it?

Reply:
no, nothing is finalised.